lib/editor/components/map/EditorMap.js

```diff
@@ -121,6 +122,15 @@ export default class EditorMap extends Component<Props, State> {
   componentDidMount () {
     window.addEventListener('resize', this._onResize)
     this.setState({willMount: false})
+
+    // From https://github.com/Leaflet/Leaflet.draw/issues/695#issuecomment-281663243
+    // Prevents creating a new leaflet draw point on drag
+    var originalOnTouch = L.Draw.Polyline.prototype._onTouch
```

Comment:
```suggestion
    const originalOnTouch = L.Draw.Polyline.prototype._onTouch
```

Reply:
This is touching the prototype of a global object, therefore, I think this should be moved outside of the component code and run once for all. (There could be an undesired side effect the second time a map mounts.)